Enhance the otelcol helm chart to support windows nodes (#190)
signalfx/splunk-otel-collector-chart: examples/README.md

splunkAccessToken: my-access-token
```

## Deploy for k8s cluster with windows worker node.

This configuration will install application for windows worker node only.

> All windows images are available in a separate `quay.io` repository for that we need to change the image pul repository to `quay.io/signalfx/splunk-otel-collector-windows`.

Reviewer: ```suggestion
All windows images are available in a separate `quay.io` repository: `quay.io/signalfx/splunk-otel-collector-windows`.
```

> Windows node only supports traces and metrics collection. Because As of now, we don’t have fluentd-hec image for the windows worker node,  as we are moving away from fluentd, No logs will be collected.

Reviewer: ```suggestion
Windows support covers traces and metrics collection only, no logs will be collected. As of now, we don’t have fluentd-hec image for the windows worker node.
```
